### ISSUE TITLE
Issue with parallel image pulls documentation

### ISSUE THREAD

**Issue by rikatz** [openshift/openshift-docs#1866]
So, on the docs, the suggested usage of parallel image pulls is as the following:

kubeletArguments:
  serialize-image-pulls:
  - true

This is a bit config, the default configuration enables the parallel image pulls? Is this safe, on an environment as RHEL7/CentOS 7 that uses Docker 1.8.2?

Also, changing the config to the suggested will not work, apparently Openshift expects an YAML with the 'true' (boolean) between quotation marks (as this is an array), so the correct would be:

kubeletArguments:
  serialize-image-pulls:
  - 'true'

Is this right? Thanks!!

File: install_config/master_node_configuration.adoc

**Comment by adellape** [comment]
> Also, changing the config to the suggested will not work

@rikatz Did you get an error? The master service restarted fine for me using the doc's syntax.

> the default configuration enables the parallel image pulls? Is this safe, on an environment as RHEL7/CentOS 7 that uses Docker 1.8.2?

The way I read it, it sounds like the default configuration uses serialized (i.e., "pull images one at a time"), as shown in the example. The <1> callout in the example is saying to change "true" to "false" to enable parallel.

cc @aveshagarwal @ncdc

**Comment by ncdc** [comment]
You risk image corruption if you set serialize-image-pulls to false with Docker prior to version 1.9.

**Comment by ncdc** [comment]
Each entry under `kubeletArguments` does require an array of strings, so doing `"true"` makes sense.

**Comment by aveshagarwal** [comment]
To enable parallel image pulls:

```
kubeletArguments:
  serialize-image-pulls:
  - "false"
```
Or

```
kubeletArguments:
  serialize-image-pulls:
  - 'false'
```

Both works. Also by default, serial image pulls is true.

**Comment by rikatz** [comment]
@adellape Yes, I get an error, as shown here by @ncdc and @aveshagarwal probably the quotation is mandatory.

Just need to fix this on the Docs :) Do you wan't me to open a PR to fix this?

Also, need to clarify that the 'true' is the default configuration, and that to change this config to 'parallel' this need to me changed to 'false'

**Comment by aveshagarwal** [comment]
@rikatz  Yes the doc needs to be corrected. Please open a PR if you can. 
Yes you are correct about the other part.
